test/workbox-precaching/sw/utils/test-addFetchListener.mjs

```diff
@@ -7,16 +7,25 @@
 */
 
 import {cacheNames} from 'workbox-core/cacheNames.mjs';
-import {addFetchListener} from 'workbox-precaching/utils/addFetchListener.mjs';
+import {addRoute} from 'workbox-precaching/addRoute.mjs';
 import {precache} from 'workbox-precaching/precache.mjs';
-import {getOrCreatePrecacheController} from 'workbox-precaching/utils/getOrCreatePrecacheController.mjs';
+import {resetDefaultPrecacheController} from '../resetDefaultPrecacheController.mjs';
 import {dispatchAndWaitForResponse} from '../../../../infra/testing/helpers/extendable-event-utils.mjs';
 
-describe(`addFetchListener()`, function() {
+
+// TODO(philipwalton): move these tests into the `../test-addRoute.mjs` after
```

Comment:
cc: @jeffposnick FYI, to make the diff easier to view for this PR, I kept these tests in this file. Once you're comfortable with the changes, I can file a separate PR to move them into the `test-addRoute` file.